Added task 1.5 and 1.6 of Teleportation Kata (#1603)
microsoft/qsharp: katas/content/teleportation/reconstruct_and_measure_message/solution.md

> The exercise can be done easily using previous solutions with slight modification.

Reviewer: ```suggestion
The exercise can be done easily using solutions to earlier exercises.
```

> - Reconstruct `qBob` qubit using the ReconstructMessage operation from exercise. This would bring the qBob state into one of the eigenstates depending to the basis mentioned.

Reviewer: ```suggestion
First, reconstruct the state of qubit `qBob` using the `ReconstructMessage` operation from an earlier exercise. This would bring the `qBob` state into one of the basis states of the given basis.
```

> - We know from Single Qubit Measurement Kata, by default all measurements are made in PauliZ basis unless specified. To get the correct result for the measurement of `qBob` qubit, perform measurement in respective basis.

Reviewer: ```suggestion
Now, we know from the "Measurements in Single-Qubit Systems" kata that the default measurement operation `M` performs measurement in the Pauli Z basis. To get the correct result for the measurement of `qBob` qubit, you need to perform the measurement in the given basis using the operation `Measure`.
```
